src/Sylius/Bundle/ApiBundle/Helpers/PromotionDataPricesTransformer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Sylius\Bundle\ApiBundle\Helpers;
+
+final class PromotionDataPricesTransformer extends AbstractDataPricesTransformer
+{
+    public function transformPricesFromData(array $data): array
+    {
+        !isset($data['rules']) ?: $data['rules'] = $this->convertPromotionRules($data['rules']);
+        !isset($data['actions']) ?: $data['actions'] = $this->convertPromotionActions($data['actions']);
+
```

Comment:
Can we use normal if condition instead if this style? 

```suggestion
        if (isset($data['rules']))
        {
            $data['rules'] = $this->convertPromotionRules($data['rules']);
        }
        
        if (isset($data['actions']))
        {
            $data['actions'] = $this->convertPromotionActions($data['actions']);
        }

```

Reply:
Theoretically that's possible xd